controllers/auth.go

```diff
@@ -459,10 +459,23 @@ func (c *ApiController) Login() {
 			}
 
 			if isSigninViaLdap {
-				user, err = object.GetUserByFields(authForm.Organization, authForm.Username)
+				ldap, err := object.GetLdap(authForm.LdapId)
+				if err != nil {
```

Comment:
If we login via ldap and
- get ldap error or 
- cannot get ldap 

may be stop the flow and return error to user?

Reply:
yes, it's good idea

Reply:
done